webcomponents/tool/src/main/frontend/js/calendar/sakai-calendar.js

```diff
@@ -160,8 +160,8 @@ export class SakaiCalendar extends LionCalendar {
         font-size: 14px;
       }
       .has-event {
-        background-color: green;
```

Comment:
You have to adjust this with the sakai-colorize-green-7 background colour, the css var. Like you did with sakai-icon above.